app/forms/csv_upload_form.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+require 'csv'
+
+class CSVUploadForm
+  include ActiveModel::Validations
+
+  attr_reader :file
```

Comment:
We are not accessing `file` outside of this class. I think we can move it to private section

Reply:
It is actually used by the validation (CSVValidator). Ex:
```ruby
  def file
    csv_form.file
  end